README.md

```diff
@@ -12,65 +12,92 @@
 
 ## Installation
 
-Install the package and all dependencies with: 
+Install the package and all dependencies with:
 
 ```r
 remotes::install_github("epiforecasts/NCoVUtils", dependencies = TRUE)
 ```
 
 ## Usage
 
-See the individual function documentation for full details of present functionality. The main function imports and cleans the linelist. 
+### Worldwide data
 
-```r
-NCoVUtils::get_linelist()
-```
+There are three sources of worldwide, country-level data on cases. One also includes deaths.
 
-Extract international case counts compiled by WHO using:
+1. Extract total global cases and deaths by country using:
+  + ```NCoVUtils::get_total_cases()```
+2. Extract international case counts compiled by the WHO using:
+  + ```NCoVUtils::get_who_cases()```
+3. Extract international case counts compiled by ECDC using:
+  + ```NCoVUtils::get_ecdc_cases()```
 
-```r
-NCoVUtils::get_who_cases()
-```
+A further function for worldwide data extracts non-pharmaceutical interventions by country:
 
-Extract international case counts compiled by ECDC using:
+* ```NCoVUtils::get_interventions_data()```
 
-```r
-NCoVUtils::get_ecdc_cases()
-```
+And anonymised international patient linelist data can be imported and cleaned with:
 
-Extract total global cases (and deaths) by country using:
+* ```NCoVUtils::get_linelist()```
 
-```r
-NCoVUtils::get_total_cases()
-```
+### Sub-national data
 
-Extract non-pharmaceutical interventions by country using:
+We currently have functions to extract sub-national level data for 11 countries. These are typically at the admin-1 level, the largest regions available. We are also working on joining the data to standard georeferencing codes to allow easy mapping.
 
-```r
-NCoVUtils::get_interventions_data()
-```
+For most countries, subnational cases can be extracted by using the following:
```

Comment:
I wouldn't say this, there is too many exceptions...
Maybe we should work on a standardized interface and/or a wrapper function that takes "country" as an argument and calls the respective function.

Reply:
> Maybe we should work on a standardized interface and/or a wrapper function that takes "country" as an argument and calls the respective function.

Added an issue for this.